resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -1801,3 +1827,12 @@ def test_report_variants(self):
             "tsv",
             output_folder / "ensembl_variants.tsv",
         )
+
+        variant_calls = VariantCall.objects.filter(
+            sample_id=ensembl_mutations.entity.id
+        )
+        self.assertEqual(len(variant_calls), 3)
+
+        variant_call_1 = variant_calls[0]
+        self.assertEqual(variant_call_1.genotype, "1/1")
+        self.assertEqual(variant_call_1.unfiltered_allele_depth, 2)
```

Comment:
This could also test the `genotype_quality` field since it is `None` in the previous example.